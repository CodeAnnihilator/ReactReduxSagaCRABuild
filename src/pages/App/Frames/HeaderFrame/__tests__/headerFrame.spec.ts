import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import HeaderFrame from '..';

describe('HeaderFrame', () => {
	it('renders the heading', () => {
		const component = shallow(HeaderFrame());
		expect(component).toMatchSnapshot('HeaderFrame');
	});
});
