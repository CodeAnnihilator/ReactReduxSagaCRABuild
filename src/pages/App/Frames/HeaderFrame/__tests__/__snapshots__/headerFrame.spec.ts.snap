// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  header
</div>
`;

exports[`HeaderFrame renders the heading: HeaderFrame 1`] = `
<div>
  header
</div>
`;
